wbooks/src/app/screens/BookDetail/styles.js

```diff
@@ -0,0 +1,77 @@
+import { StyleSheet } from 'react-native';
+
+import { cyan, lightGray, red, white, lightBlue, lightGreen } from '../../../constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: cyan
+  },
+  cardDetail: {	
+    borderRadius: 5,
+    backgroundColor: white,
+    margin: 20
+  },
+  cardContainer: {
+    flexDirection: 'row',
+    borderRadius: 5,
+    marginHorizontal: 30,
+    marginTop: 8
+  },
+  image: {
+    width: 69,
+    height: 105,
+    marginStart: 18,
+    marginTop: 7
+  },
+  textContainer: {
+    flex: 1,
+    marginHorizontal: 20,
+    borderTopWidth: 0
```

Comment:
is this necessary? Can't you just delete it?

Reply:
done